hack/metadata.schema.json

```diff
@@ -1,114 +1,735 @@
+                                                            "$id": "#/properties/rbac/properties/roles/items/anyOf/0/properties/rules/items/anyOf/0/properties/resources",
+                                                            "type": "array",
+                                                            "title": "The resources schema",
+                                                            "description": "An explanation about the purpose of this instance.",
+                                                            "default": [],
+                                                            "examples": [
+                                                                [
+                                                                    "pods"
+                                                                ]
+                                                            ],
+                                                            "additionalItems": true,
+                                                            "items": {
+                                                                "$id": "#/properties/rbac/properties/roles/items/anyOf/0/properties/rules/items/anyOf/0/properties/resources/items",
+                                                                "anyOf": [
+                                                                    {
+                                                                        "$id": "#/properties/rbac/properties/roles/items/anyOf/0/properties/rules/items/anyOf/0/properties/resources/items/anyOf/0",
+                                                                        "type": "string",
+                                                                        "title": "The first anyOf schema",
+                                                                        "description": "An explanation about the purpose of this instance.",
+                                                                        "default": "",
+                                                                        "examples": [
+                                                                            "pods"
+                                                                        ]
+                                                                    }
+                                                                ]
+                                                            }
+                                                        },
+                                                        "resourceNames": {
+                                                            "$id": "#/properties/rbac/properties/roles/items/anyOf/0/properties/rules/items/anyOf/0/properties/resourceNames",
+                                                            "type": "array",
+                                                            "title": "The resourceNames schema",
```

Comment:
Need to make this optional

Reply:
this schema is auto generated, and I made no changes to that, this is just for testing, @qhua948 Could you update the existing one that already in the repo?

Reply:
:+1: